src/foam/core/Property.js

```diff
@@ -522,6 +523,7 @@ foam.CLASS({
           if ( assertValue ) assertValue.call(this, newValue, prop);
 
           if ( preSet ) newValue = preSet.call(this, oldValue, newValue, prop);
+          if ( trim ) newValue = newValue.trim();
```

Comment:
I think this should go before adapt and assertValue. Do you agree?

Reply:
Yes

Reply:
Sorry for the retro review here, but why have trim baked into foam.core.Property? Isn't this just a String concept? We generate an adapt on String that should do this work.

Reply:
See: https://github.com/foam-framework/foam2/pull/2106/files